stanford_video.views_default.inc

```diff
@@ -152,13 +152,6 @@ function stanford_video_views_default_views() {
   $handler->display->display_options['fields']['views_bulk_operations']['vbo_settings']['force_single'] = 0;
   $handler->display->display_options['fields']['views_bulk_operations']['vbo_settings']['entity_load_capacity'] = '10';
   $handler->display->display_options['fields']['views_bulk_operations']['vbo_operations'] = array(
-    'action::node_assign_owner_action' => array(
```

Comment:
Those actions are not available in the View when installed on a clean Drupal. So there's either an unmet dependency, or the syntax for these actions has changed. Yarr.